v3/lints/apple/lint_ct_sct_policy_count_unsatisfied.go

```diff
@@ -33,15 +33,16 @@ func init() {
 		Citation:      "https://support.apple.com/en-us/HT205280",
```

Comment:
From here on out it is (thankfully) pretty repetitive. It makes a new constructor for every type, deletes `Initialize`, and replaces `Lint: &structName{}` with `Lint: NewStructName`.